solutions/src/main/java/ru/sberbank/school/task02/ExtendedFxConverter.java

```diff
@@ -0,0 +1,20 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public class ExtendedFxConverter implements ExtendedFxConversionService {
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol, BigDecimal amount, Beneficiary beneficiary) {
```

Comment:
Line is longer than 120 characters (found 135).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-337441515>*